www/front_src/src/components/Hook/index.tsx

```diff
@@ -44,11 +44,7 @@ const LoadableHooks = ({
 
 const Hook = React.memo(
   (props: Props) => {
-    return (
-      <Suspense fallback={null}>
-        <LoadableHooks {...props} />
-      </Suspense>
-    );
+    return <LoadableHooks {...props} />;
```

Comment:
What happens here, when a module such as BAM is installed, the page is refreshed and the top counter hook is not yet loaded?